packages/config/config/devices/0x0345/templates/swidget_template.json

```
@@ -0,0 +1,1504 @@
+		"description": "Used to set the power value for Power 1 (the controlled load) that indicates exit from a Power 1 High state.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 2400
+	},
+	"power_1_low_exit_value": {
+		// param 4
+		"label": "Power 1 Low Exit Value",
+		"description": "Used to set the power value for Power 1 (the controlled load) that indicates exit from a Power 1 Low power state.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_high_delay": {
+		//param 5
+		"label": "Power 1 High Delay",
+		"description": "Used to set the time duration that Power 1 High state change power level conditions must exist before Power 1 High state entry and/or exit is registered, and the associated actions are executed.",
+		"unit": "seconds",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_low_delay": {
+		//param 6
+		"label": "Power 1 Low Delay",
+		"description": "Used to set the time duration that Power 1 Low state change power level condition must exist before Power 1 Low state entry and/or exit is registered, and the associated actions are executed.",
+		"unit": "seconds",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_periodic_report": {
+		// param 7
+		"label": "Power 1 Periodic Report",
+		"description": "Used to set the time in seconds between sending Power 1 METER REPORTs to the Lifeline.",
+		"unit": "seconds",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 3600
+	},
+	"power_1_change_report": {
+		// param 8
+		"label": "Power 1 Change Report",
+		"description": "Used to set the power in Watts that a reported reading of Power 1 must change by since the last reported value before a new METER REPORT is sent to the Lifeline.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 10
+	},
+	"power_1_renotification_delay": {
+		// param 9
+		"label": "Power 1 Renotification Delay",
+		"description": "Used to set the time in seconds between sending repeated Power 1 METER REPORTs to the Lifeline while in a Power 1 Low or High state.",
+		"unit": "seconds",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_high_entry_set_value": {
+		// param 10
+		"label": "Power 1 High Entry SET value",
+		"description": "BASIC SET value sent to associated nodes on entry into Power 1 High state.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 255
+	},
+	"power_1_high_exit_set_value": {
+		// param 11
+		"label": "Power 1 High Exit SET value",
+		"description": "BASIC SET value sent to associated nodes on exit from Power 1 High state.",
```

```suggestion
		"label": "Load High Power: Basic Set Value (Exit)",
```
and similar for the others